x11-plugins/pidgin-skypeweb/pidgin-skypeweb-1.5.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="SkypeWeb Plugin for Pidgin"
+HOMEPAGE="https://github.com/EionRobb/skype4pidgin"
+SRC_URI="https://github.com/EionRobb/skype4pidgin/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3+"
```

Comment:
I just repushed those changes in the branch.

I’m not particularly willing to co-mainain this package as I only occasional user. I dug into it because of login issues.

Reply:
Above LICENSE is what I meant.